apps/trading/components/rewards-container/active-rewards.tsx

```diff
@@ -0,0 +1,530 @@
+              </span>
+              {
+                <Tooltip
+                  description={t(
+                    DistributionStrategyDescriptionMapping[
+                      dispatchStrategy.distributionStrategy
+                    ]
+                  )}
+                  underline={true}
+                >
+                  <span className="text-xs">
+                    {
+                      DistributionStrategyMapping[
+                        dispatchStrategy.distributionStrategy
+                      ]
+                    }
+                  </span>
+                </Tooltip>
+              }
+            </div>
+
+            <div className="flex flex-col gap-2 items-center text-center">
+              <CardIcon
+                iconName={VegaIconNames.LOCK}
+                tooltip={t(
+                  'Number of epochs after distribution to delay vesting of rewards by'
+                )}
+              />
+              <span className="text-muted text-xs whitespace-nowrap">
+                {t('{{lock}} epochs', {
+                  lock: kind.dispatchStrategy?.lockPeriod,
+                })}
+              </span>
+            </div>
+          </div>
+
+          <span className="border-[0.5px] border-gray-700" />
+          {/* TODO use market symbol or market name */}
+          <span>
+            {DispatchMetricLabels[dispatchStrategy.dispatchMetric]}
+            {marketName ? ` • ${marketName}` : ` • ${dispatchAsset?.name}`}
+          </span>
+
+          <div className="flex items-center gap-8 flex-wrap">
+            {kind.endEpoch && (
+              <span className="flex flex-col">
+                <span className="text-muted text-xs">{t('Ends in')}</span>
+                <span>
+                  {t('{{epochs}} epochs', {
+                    epochs: kind.endEpoch
+                      ? formatNumber(kind.endEpoch - currentEpoch)
+                      : '-',
+                  })}
+                </span>
+              </span>
+            )}
+
+            {
+              <span className="flex flex-col">
+                <span className="text-muted text-xs">{t('Assessed over')}</span>
+                <span>
+                  {dispatchStrategy.windowLength === 1
+                    ? t('{{epochs}} epoch', {
+                        epochs: formatNumber(dispatchStrategy.windowLength),
+                      })
+                    : t('{{epochs}} epoch(s)', {
+                        epochs: formatNumber(dispatchStrategy.windowLength),
+                      })}
+                </span>
+              </span>
+            }
+          </div>
+
+          {dispatchStrategy?.dispatchMetric && (
+            <span className="text-muted text-sm h-[2rem]">
+              {t(DispatchMetricDescription[dispatchStrategy?.dispatchMetric])}
+            </span>
+          )}
+
+          <span className="border-[0.5px] border-gray-700" />
```

Comment:
Can you use clight and cdark. Also this can just be border-b for 1px

Reply:
with border-b and vega clight/cdark it would look more prominent and I was trying to not do that to match the design
<img width="531" alt="Screenshot 2023-12-27 at 18 01 10" src="https://github.com/vegaprotocol/frontend-monorepo/assets/16125548/d6300cef-bc05-47ba-ac29-be07751373e7">
<img width="447" alt="Screenshot 2023-12-27 at 18 01 31" src="https://github.com/vegaprotocol/frontend-monorepo/assets/16125548/2287cfe9-ceed-4b96-bb33-225dac1641fe">